src/cli/README_NETDATA.md

```diff
@@ -379,6 +379,15 @@ Service entries are listed under `Services` header:
 - Context ID
 - Compress flag (`c` if marked or `-` otherwise).
 
+Commissioning Dataset information is printed under `Commissioning` header:
```

Comment:
Please also update the CLI doc comments with this information https://github.com/openthread/openthread/blob/main/src/cli/cli_network_data.cpp#L753

Reply:
Thanks @Vyrastas. I think this is already included/changed in  the PR. Is there maybe anywhere else I may have missed?

Reply:
Ah my fault, missed that.  One comment there and we're good